setup.py

```diff
@@ -115,6 +115,7 @@ def write_version(filename=os.path.join(*['airflow',
 dask = [
     'distributed>=1.15.2, <2'
     ]
+databricks = ['requests>=2.5.1, <3']
```

Comment:
does other Airflow integrator that relies on rest API also uses requests package, and have the same requirement?